src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  background: $color-white;
+  border: 1px solid #e5e5e5;
+  border-radius: 5px;
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  
+  &:not(:nth-child(4n)) {
+    margin-right: $margin-right;
+  }
+
+  &:not(:nth-child(-n+4)) {
+    margin-top: 46px;
+  }

```

Comment:
Do not set margins directly to the BEM block. Use BEM mixes instead

Reply:
changed